src/Website/Resources/ordercancelemail.php

```diff
@@ -1,21 +1,30 @@
 <?php
 require 'PHPMailerAutoload.php';
-
-$mail = new PHPMailer;
+require 'logindata.php';
 
 //$mail->SMTPDebug = 3;                               // Enable verbose debug output
 
+
+
+
+$mail = new PHPMailer(true);
```

Comment:
is this a change in how PHPMailer works or is it because of the addition of separating out the login data?

Reply:
gmail needs this in order to work. it is what was missing the first time I tried...